main.go

```diff
@@ -1157,7 +1157,11 @@ func (tf *transformer) findReflectFunctions(files []*ast.File) {
 	}
 }
 
+// cmd/bundle will include a go:generate directive in its output by default.
+// Ours specifies a version and doesn't assume bundle is in $PATH, so drop it.
+
 //go:generate go run golang.org/x/tools/cmd/bundle@v0.1.9 -o cmdgo_quoted.go -prefix cmdgoQuoted cmd/internal/quoted
+//go:generate sed -i /go:generate/d cmdgo_quoted.go
```

Comment:
Does windows necessarily have sed?

Otherwise we should probably make the above a go get and keep the autogenerated line in the other file.

Reply:
sed is probably not portable, but does that really matter in practice? This only needs to run on the machine that runs `go generate`, not for all users.

I really dislike having `go generate` run `go install`, because that messes with the user's system. I'd much rather not depend on `$PATH` at all.

A better fix would be to teach `bundle` how to not produce the line, which it only supports now via stdout, but we can't use that via `go:generate` without a shell. I can think of some enhancement proposals, but I fear I'd be going down a rabbit hole :)

If you're bothered by the portability, then I think I'll just not make this change - the duplicate `go:generate` is wasteful, but at least it doesn't run `go install`.

Reply:
Right I think we are all not on windows anyway